benchmark/citation/appnp.py

```diff
@@ -20,6 +20,9 @@
 parser.add_argument('--normalize_features', type=bool, default=True)
 parser.add_argument('--K', type=int, default=10)
 parser.add_argument('--alpha', type=float, default=0.1)
+parser.add_argument('--inference', type=bool, default=False)
+parser.add_argument('--profile', type=bool,
+                    default=False)  # Currently support profile in inference
```

Comment:
```suggestion
parser.add_argument('--profile', action='store_true')
```
And in remaining scripts.

Reply:
Done.